pages/docs/running-grouparoo.mdx

```diff
@@ -1,8 +1,72 @@
 ---
 title: "Running Grouparoo"
 date: "2021-02-02"
-section: "config"
-pullQuote: "Section coming soon ..."
+section: "running-grouparoo"
+pullQuote: "There are a number of ways to run Grouparoo to fit your workflow."
 ---
 
-Section coming soon ...
+There are a number of ways to run Grouparoo to fit your workflow. We believe that the 3 phases to a successfully Grouparoo Deployment are:
+
+1. Project Maintenance
+2. Configuration
+3. Development, Testing, and Deployment
+
+### Project Maintenance
+
+Configuring your Grouparoo Project starts by making a new `git` repository in a new directory. You then can use the `grouparoo` cli tool to initialize a new project.
+
+Grouparoo is a node.js project at heart, so you will be configuring your environment, creating a `package.json` and installing dependencies. You can also use commands like `npm install` to replace the Grouparoo commands in this section.
+
+<Button href="/docs/running-grouparoo/commands">
+  Learn about the Grouparoo CLI
+</Button>
+
+### Configuration
+
+To use Grouparoo to import and export your data, you will be configuring our main objects: Apps, Sources, Destinations, Properties, etc. You can [learn more about Grouparoo's product concepts here](http://localhost:3001/docs/getting-started/product-concepts). You will be using the Grouparoo CLI's `generate`, `validate`. and `apply` commands to ensure that the application can read from your Sources and write to your Destinations.
+
+Configuration files are either JSON or Javscript which allow you the flexibility to manipulate data or load configuration from external sources:
+
+```js
+// file: config/apps/data_warehouse.js
+
+exports.default = async function GetDataWarehouseConfig() {
+  const { databasePassword } = await fetch(`/api/company/secrets`);
+
+  return [
+    {
+      name: "Data Warehouse",
+      class: "app",
+      id: "data_warehouse",
+      type: "postgres",
+      options: {
+        host: "127.0.0.1",
+        port: "5432",
+        username: "person",
+        password: databasePassword,
+        database: "data_warehouse",
+      },
+    },
+  ];
+};
+```
+
+You can learn more about the [configuration files here](/docs/deployment/code-config).
+
+<Button href="/docs/running-grouparoo/commands#generate">
+  Learn about the Generate Command
+</Button>
+
```

Comment:
We had broken out Configuration as its own section so the docs flowed like a narrative for someone setting up Grouparoo:

1. Install/Deploy
2. Configure
3. Run

This whole section was originally meant to make the assumption that you were already configured. I think it's a good idea to still _mention_ configuration, but point to the config section for doing so.

(I'm thinking you'd also put validate and apply in the config section.)

That said, reworking the config section is a whole other story that I was planning to pick up, so I'm totally fine if we keep this here for now and I pull from it for now and I move it when I pick that story up today/Monday.

I updated some notes as a result of this comment [in the doc](https://docs.google.com/document/d/1iOzxhJzRdnL4lnkNwb1Daqld1HXjy9ORU8CZmkgwm4c/edit).